invoicing_app/tests/tests.py

```diff
@@ -169,12 +174,57 @@ def test_localize_date_old(self):
         my_date = dt(2020, 4, 11, 0, 0)
         self.assertEqual(
             str(self.my_command.localize_date('AR', my_date)),
-            "2020-04-11 00:00:00-03:54"
+            '2020-04-11 00:00:00-03:54'
+        )
+        self.assertEqual(
+            str(self.my_command_new.localize_date('AR', my_date)),
```

Comment:
Nit: Consider moving the `localize_date` helper out of the commands and to a different module? It could be tested separately and imported within both commands, instead of running the exact same of assertions against the copies in each.